apps/src/applab/applab.js

```diff
@@ -200,6 +203,20 @@ Applab.makeFooterMenuItems = function(isIframeEmbed) {
       newWindow: true
     }
   ].filter(item => item);
+
+  var userAlreadyReportedAbuse =
+    cookies.get('reported_abuse') &&
+    _.includes(
+      JSON.parse(cookies.get('reported_abuse')),
+      project.getCurrentId()
+    );
+
+  if (userAlreadyReportedAbuse) {
+    _.remove(footerMenuItems, function(menuItem) {
+      return menuItem.key === 'reportAbuse';
```

Comment:
this is spelled `'report-abuse'` in 2 places below. is this a typo here? please make sure this is working as expected in applab.